pwndbg/dbg/lldb/__init__.py

```diff
@@ -119,6 +121,12 @@ def reg_write(self, name: str, val: int) -> bool:
         if val < 0:
             raise RuntimeError("Tried to write a register with a negative value")
 
+        # Writing to the PC using the normal register write flow causes the
+        # inner object to be automatically invalidated by LLDB, so we have to
+        # handle jumps manually using SBFrame::SetPC.
+        if name.lower() == reg_sets[pwndbg.aglib.arch.name].pc:
```

Comment:
is it okay that we are using lower only here and not below when we do rename_register?